packages/application-generic/src/services/cache/key-builders/shared.ts

```diff
@@ -32,6 +32,7 @@ export enum CacheKeyPrefixEnum {
   INTEGRATION = 'integration',
   ENVIRONMENT_BY_API_KEY = 'environment_by_api_key',
   GROUPED_BLUEPRINTS = 'grouped-blueprints',
+  USER_DATA = 'user_data_env_entity_user_entity',
```

Comment:
```suggestion
  AUTH_SERVICE = 'auth_service',
```
Mainly because it is not our User data entity and we try to return multiple stuff, all related to a certain service.

Reply:
awesome suggestion IMO thanks!